pyspectral/rayleigh.py

```diff
@@ -260,10 +260,12 @@ def get_reflectance(self, sun_zenith, sat_zenith, azidiff, bandname, redband=Non
             if redband is not None:
                 redband = from_array(redband, chunks=redband.shape)
 
-        clip_angle = rad2deg(arccos(1. / sunz_sec_coord.max()))
+        clip_angle = nan_to_num(rad2deg(arccos(1. / sunz_sec_coord.max())))
```

Comment:
you have duplicated lines here, maybe a small function would be nice? Maybe even the docstring of that function could say that the angles are filled with 0s?

Reply:
Yes, good point. I have a proposal coming.

Reply:
Okay now @mraspaud ?